### DISCUSSION TITLE
Add sample rate changing directly in the drop down menu

### DISCUSSION THREAD

**Discussion by Justinwang677** [vincentneo/LosslessSwitcher#26] [Ideas]
Some app don't change automatically so it would be a good idea to add a sample rate changing directly in the drop down menu and also bit depth option too

**Comment by vincentneo** [comment]
Oh you mean like a quick way to override sample rate? Rather than going into Audio MIDI Setup app?

**Reply by Justinwang677** [reply]
Yeah like if an app isn't already supported users can go into the menu and select there own sample rate without having to go into the midi app

**Reply by vincentneo** [reply]
alright got it!

**Reply by tomekbuszewski** [reply]
Great idea, it would be much more convenient than going into MIDI Setup!
